core/renderers/common/cursor_svg.js

```diff
@@ -452,9 +411,9 @@ Blockly.CursorSvg.prototype.positionPrevious_ = function(width, cursorOffset, cu
  * @param {number} y The new y, in workspace units.
  * @param {number} width The new width, in workspace units.
  * @param {number} height The new height, in workspace units.
- * @private
+ * @protected
```

Comment:
Should the other position methods also become protected?  positionPrevious, positionLine, etc.

Reply:
I think positionLine, but positionPrevious and positionInput are more specific to the certain renderer. For example both input and output use the puzzle_tab constants.